lib/logstash/outputs/http.rb

```diff
@@ -16,12 +22,11 @@ class LogStash::Outputs::Http < LogStash::Outputs::Base
   # URL to use
   config :url, :validate => :string, :required => :true
 
-  # validate SSL?
-  config :verify_ssl, :validate => :boolean, :default => true
+  # DEPRECATED. Set 'ssl_certificate_validation' instead
+  config :verify_ssl, :validate => :boolean, :default => true, :deprecated => true
```

Comment:
deprecated can be a string describing why and what can be done to update: https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/lib/logstash/outputs/elasticsearch.rb#L84